templates/default/sql/dml/undo/1.4.0__undo.sql.erb

```diff
@@ -1,8 +1,6 @@
 DELETE FROM `hopsworks`.`variables` WHERE `id`='yarn_runtime';
 REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("anaconda_dir", "<%= node['conda']['base_dir'] %>");
 DELETE FROM `hopsworks`.`variables` WHERE `id`='docker_mounts';
-DELETE FROM `hopsworks`.`variables` WHERE `id`='docker_registry_name';
-DELETE FROM `hopsworks`.`variables` WHERE `id`='docker_registry_port';
```

Comment:
Shouldn't the undo put back the values?
```
REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_name", "<%= @registry_host %>");	
REPLACE INTO `hopsworks`.`variables` (`id`, `value`) VALUES ("docker_registry_port", "<%= node['hops']['docker']['registry']['port'] %>");
```

Reply:
No, we are removing the fact that the values are being added to the database completely (adding the value has been done in the same "release" as we are now) so it will be as if these values had never existed.